### ISSUE TITLE
Got 65.67% accuracy 

### ISSUE THREAD

**Issue by Ideal-Bai** [phamquiluan/ResidualMaskingNetwork#9]
sorry, I tried to run the main_fer2013.py file and specified the model name is resmasking_dropout1, but I only got 65.63% accuracy on the private test dataset. What caused this?

**Comment by phamquiluan** [comment]
refer to #7

**Comment by Ideal-Bai** [comment]
Thank you so much.

**Comment by phamquiluan** [comment]
are you successfully reproduce it ? 
@Ideal-Bai

**Comment by jet-yangqs** [comment]
@phamquiluan 
I follow your training steps to train the model on FER2013 dataset,  but I get the accuracy no better than 73.6%. I use the pretrained resnet34 refer to #7  and tried to replace the "arch" in "fer2013_config.json" file with "resmasking_dropout1", "resmasking_dropout2", "resmasking_dropout3", "resmasking_dropout4". Can you give me some advice to reproduce your result? thanks a lot!

**Comment by gaoruiyang** [comment]
Cannot reproduce 74% yet

**Comment by zhuning123-bur** [comment]
> 还不能重现 74%

Did you achieve it？
